syntax = "proto3";

package showcollectionservice;

// define the service
service ShowCollectionService {
 rpc SendShowCollection (ShowCollectionRequest) returns (ShowCollectionResponse) {}
}

// request showcollection
message ShowCollectionRequest {
    string showslug = 1;
}

//response showcollection
message ShowCollectionResponse{
    string showid = 1;
    bool success = 2;
}
